test/client/fixtures/automation/regression-test.js

```diff
@@ -339,33 +339,31 @@ $(document).ready(function () {
         });
     });
 
-    asyncTest('B237672 - TesCafe throw exception "Access is denied" after trying to get content of iframe in IE browsers', function () {
-        let clicked = false;
+    asyncTest('B237672 - TesCafe does not throw exception "Access is denied" after trying to get content of iframe', function () {
```

Comment:
Let's do more test improvements:
* rename to `B237672 - TesCafe should not throw an exception "Access is denied" on accessing to a content of the across-domain iframe`
* `.attr('src', 'http://www.cross.domain.com')` - use any test page instead of a public url
* rename `const iframeBody = ...` to `iframeDocument`
* add a clear message for `throw new Error();` statement (e.g. `Click handler for a cross-domain iframe should not be called`).